tests/test_data.py

```diff
@@ -460,5 +460,21 @@ def test_get_pulse_times(self):
         self.assertTrue(np.all(test_03_times == pulse_times))
         self.assertTrue(np.all(test_03_dc == pulse_dc))
 
+class Backup(unittest.TestCase):
+
+    def test_move_files(self):
+        source_dir = [['tests/data/test_source/dir_1'],['tests/data/test_source/dir_2']]
+        #source_files = [os.path.join(source_dir,'load_hdf_contents_test.hdf'), os.path.join(source_dir,'load_hdf_group_test.hdf')]
+
+        dest_dir = 'tests/tmp/test_destination'
+        if not os.path.exists(dest_dir):
+            os.mkdir(dest_dir)
+
+        move_files(source_dir,dest_dir)
+        dest_dir_file_bool1 = os.path.exists('tests/tmp/test_destination/dir_1')
+        dest_dir_file_bool2 = os.path.exists('tests/tmp/test_destination/dir_2')
+        self.assertTrue(dest_dir_file_bool1)
+        self.assertTrue(dest_dir_file_bool2)
```

Comment:
also check if the files themselves have been copied, e.g. `self.assertTrue(os.path.exists('tests/tmp/test_destination/dir_1/load_ecube_data.png'))`